core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.ts

```diff
@@ -496,15 +497,24 @@ export class StoryNodeEditorComponent implements OnInit, OnDestroy {
   updateLocalEditableOutline($event: string): void {
     if (this.editableOutline !== $event) {
       this.editableOutline = $event;
-      if (!this.chapterOutlineButtonsAreShown && $event) {
-        this.toggleChapterOutlineButtons();
-      }
       this.changeDetectorRef.detectChanges();
     }
   }
 
-  toggleChapterOutlineButtons(): void {
-    this.chapterOutlineButtonsAreShown = !this.chapterOutlineButtonsAreShown;
+  onSaveButtonClicked(): void {
+    this.outlineEditViewIsShown = $(this.oldOutline).text();
```

Comment:
Why do you need this line at all? Also, I think that outlineEditViewIsShown should be a boolean and not a string since the name indicates true/false...

Additionally I am confused by the type of this.oldOutline -- it seems like it's supposed to be a string so I'm not sure why you're interpreting it using jQuery. Are you trying to check whether it's empty? If so, would `!this,oldOutline` suffice?

Reply:
I used jQuery because  `outlineEditViewIsShown` is used to display the saved text as shown below.
Also html tag is ramained without jQuery.
![remain_html_tag](https://github.com/oppia/oppia/assets/140389350/95f49efe-e1e4-449f-bb07-b163d2edd10a)